src/signalo/core/models.py

```diff
@@ -40,3 +41,37 @@ class Sign(ComputedFieldsModel):
     )
     def geom(self):
         return self.pole.geom
+
+
+@register_oapif_viewset()
+class Sign(ComputedFieldsModel):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    sign_type = models.ForeignKey(
+        OfficialSignType,
+        models.SET_NULL,
+        blank=True,
+        null=True,
+        related_name="official_sign",
+    )
+    azimuth = models.ForeignKey(
+        Azimuth,
+        models.SET_NULL,
+        blank=True,
+        null=True,
+        related_name="signs",
+    )
+    order = models.IntegerField(null=False, blank=False)
+
+    @computed(
+        models.PointField(
+            srid=settings.GEOMETRY_SRID,
+            verbose_name=_("Geometry"),
+            null=True,
+        ),
+        depends=[
+            ("self", ["azimuth"]),
+            ("azimuth", ["pole"]),
```

Comment:
looking at the other definition
```suggestion
            ("azimuth", ["geom"]),
```